src/map-func.mjs

```diff
@@ -0,0 +1,9 @@
+import curry from './internal/curry'
+import map from './map'
+import zip from './zip'
+
+function mapFunc (functions, inputs) {
+  return map(([func, input]) => func(input), zip(functions, inputs))
```

Comment:
Wouldn't a better implementation be:
```js
merge((item, func) => func(item), [[...items], [...funcs]])
```
And being rather terse (and also fairly descriptive) wouldn't it be better to use this implementation in code should this presumably rather rare need arise?

Reply:
OK, I've just taken a careful look at merge, and it isn't at all what I thought it was.